base/libgit2/commit.jl

```diff
@@ -15,7 +15,8 @@ function message(c::GitCommit, raw::Bool=false)
     if msg_ptr == C_NULL
         return nothing
     end
-    return unsafe_string(msg_ptr)
+    msg_str = Base.@gc_preserve c unsafe_string(msg_ptr)
```

Comment:
This is fine for the compiler as is, but probably best to wrap the entire function (from before the ccall to after the unsafe_string) in `gc_preserve c` to make sure we get the scope correct.

Reply:
OK, I'll go in and fix these. Thanks for the assist!